tests/open_alchemy/test_build.py

```diff
@@ -491,10 +491,27 @@ def test_calculate_version(spec, spec_str, expected_version):
     assert returned_version == expected_version
 
 
+@pytest.mark.parametrize(
+    "package_format",
+    [
+        pytest.param(None, id="build without package"),
+        pytest.param("sdist", id="build with a sdist package"),
+        pytest.param(
+            "wheel",
+            id="build with a wheel package",
+            marks=pytest.mark.xfail(reason="the wheel package is not installed"),
```

Comment:
Could this particular problem be resolve by making the installation also include the `wheel` package?

The additional request I would have is to add a test for when the wheel package is not installed. There should be some examples for when `PyYAML` is not installed in the code based integration tests.